.github/workflows/test.yml

```diff
@@ -268,3 +249,25 @@ jobs:
         with:
           targets: lint,build,test
           projects: ${{matrix.projectName}}
+
+  validate_swagger:
+    name: Validate Swagger
+    runs-on: ubuntu-latest
+    needs: [get-affected]
+    if: ${{ fromJson(needs.get-affected.outputs.test-unit)[0] }}
```

Comment:
We would expect to validate the Swagger when something affecting the unit tests for the API change.